vignettes/getting-started.Rmd

```diff
@@ -205,7 +205,54 @@ This code uses `purrr::map_lgl` to filter the run log to only rows containing a
 ```{r filtering}
 log_df %>% 
   filter(purrr::map_lgl(tags, ~ "2 compartment" %in% .x)) %>%
-  select(run_id, description, tags, decisions)
+  select(absolute_model_path, description, tags, decisions)
+```
+
+## Checking if models are up-to-date with config_log()
+
+When `babylon` runs a model, it creates a file named `bbi_config.json` in the output directory. This file contains a lot of information about the state and configuration at runtime. Notably, it contains an md5 digest of both the model file _and_ the data file at run time. Users can compare this to the _current_ md5 hashes of these two files to check if any model or data files have changed _since the model was last run_. This serves as a check that the outputs are up-to-date with the model and data.
+
+Users can call `config_log` directly, but it is likely more useful to use `run_log() %>% add_config()` to join the two together automatically.
+
+```{r add_config fake prep, include=FALSE}
+# read in original json (which was actually fake a.k.a. copied from somewhere else)
+orig_json <- jsonlite::fromJSON(file.path(getOption("rbabylon.model_directory"), "1", "bbi_config.json"))
+
+# fix data path (just in case)
+orig_json$data_path <- stringr::str_replace(orig_json$data_path, "/data/", "/extdata/")
+
+# copying the outputs from the original run 
+# and setting model md5 to match control stream (remember this "original" is fake)
+for (i in c("1", "2", "3")) {
+  out_dir <- file.path(getOption("rbabylon.model_directory"), i)
+  if (!fs::dir_exists(out_dir)) fs::dir_create(out_dir)
+  new_json <- orig_json
+  new_json$model_md5 <- tools::md5sum(ctl_ext(out_dir))
+  readr::write_file(jsonlite::toJSON(new_json), file.path(out_dir, "bbi_config.json"))
+}
+```
+
+```{r add_config}
+log_df <- run_log() %>% add_config()
+log_df
+```
+
+Then a user can easily check the md5's digests from the config log against those of the current files.
+
+```{r check config md5}
+# extract path to data files and model files
+norm_data_paths <- fs::path_norm(file.path(log_df$absolute_model_path, log_df$data_path))
+norm_model_paths <- get_model_path(log_df)
+
+# get md5 digests and compare to those from config_log() columns
+log_df <- log_df %>% mutate(
```

Comment:
I'm all for building this in as a helper function for future releases, seems like a common enough application of the hash